apps/chat/src/pages/index.tsx

```diff
@@ -115,6 +116,7 @@ function Home() {
             {isReplaceModalOpened && (
               <ReplaceConfirmationModal isOpen={isReplaceModalOpened} />
             )}
+            <RenameConversationModal />
```

Comment:
I guess we can render it inside `<ConversationContextMenu />`, like it's done for publishing, etc. And we can store id of renaming entity inside component state and don't overload conversation reducer